packages/thirdweb/package.json

```diff
@@ -166,8 +166,18 @@
     "viem": "2.10.9"
   },
   "peerDependencies": {
+    "@aws-sdk/client-lambda": "3.549.0",
+    "@aws-sdk/credential-providers": "3.549.0",
+    "amazon-cognito-identity-js": "^6.3.12",
+    "aws-amplify": "^5.3.18",
```

Comment:
all of these *have* to be marked as optional below in `"peerDependenciesMeta"` otherwise package managers will report them as missing peer dependencies in projects where people don't also have the adapter installed (which we do not want)